### PR TITLE
Language packages from DataverseEU repository

### PR DESCRIPTION
**Pull request by 4tikhonov**
I've added a notes about supported version of Dataverse as some of translations not up-to-date yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # dataverse-language-packs
 Repository for language files associated with Dataverse
+
+Some language packages supported by Dataverse version not older than 4.9.4 (please see notes below)
+
+Available languages:
+- English
+- [French (Canada)](https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/tree/develop/fr_CA) maintained by [Scholars Portal](https://dataverse.scholarsportal.info)
```
Comment (mhvezina) [inline README.md:8]: French (Canada) translation is done by Bibliothèques Université de Montréal, in collaboration with Scholars Portal. If anyone wants to comment about/suggest translations to us.

**Comment by amberleahey** [comment]
@4tikhonov you now have permission to merge this pull request. We can accept the readme changes.

**Comment by amberleahey** [comment]
As new translators join we can give them permission to make edits and pull requests.

File: README.md (hunk 1 above)
Comment (amberleahey) [reply README.md:8]: sounds good, @4tikhonov can you add this statement?

**Comment by 4tikhonov** [comment]
Thanks, @amberleahey! I'm merging now.

**Pull request merged**